### ISSUE TITLE
Performance of fine-tuning on UCF101

### ISSUE THREAD

**Issue by zhihuilics** [kenshohara/3D-ResNets-PyTorch#24]
I downloaded the network ResNet-101 pretrained on Kinetics, and fine-tuned on UCF101 following the example script. However, I can only get 82.5 by averaging the three splits. In the paper, the authors reported 88.9.  Any suggestion?

**Comment by zhihuilics** [comment]
I also downloaded the network ResNext-101 pretrained on Kinects, and fine-tuned on UCF101 following the example script. I can only get 86.0 instead of 90.7. Any suggestion?

**Comment by chandnikathuria1992** [comment]
i have not experimented with code till now...and have to do....but i think pre-trained model in the repository is for split-1 only. But u are checking for accuracy for three splits over which model is not pre-trained. is it?

**Comment by zhihuilics** [comment]
No, I don't think so. The pre-trained model was trained on Kinetics. I fine-tuned on the three splits and averaged the results. The authors also released their fine-tuned model on split-1. I will test that model now.

**Comment by chandnikathuria1992** [comment]
Ok..I thought u are using Fine-tuned one.


On 23 Jan 2018 10:43 pm, "zhihuilics" <notifications@github.com> wrote:

No, I don't think so. The pre-trained model was trained on Kinetics. I
fine-tuned on the three splits and averaged the results. The authors also
released their fine-tuned model on split-1. I will test that model now.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/kenshohara/3D-ResNets-PyTorch/issues/24#issuecomment-359862357>,
or mute the thread

**Comment by zhihuilics** [comment]
Tried the fine-tuned one on UCF101-split 1. Still can only get 87.9, one point less than 88.9.

**Comment by chandnikathuria1992** [comment]
U have fine tuned in same way ie. conv5x and Fc layers for all splits or tried something else?
have u tried to increase the number of epochs ?

**Comment by kenshohara** [comment]
@zhihuilics 
Thank you for trying my code!
Could you tell me more details?
You got 87.9% accuracy by using resnext-101-kinetics-ucf101_split1.pth?
Could you tell me your options when you execute the testing?

**Comment by mariankyoussef** [comment]
I also got 80.3% when fine tuning resnet-50 on UCF-101 split-1 instead of 89%, my settings are:

python main.py --root_path ~/big/3D-ResNets-PyTorch --video_path ~/big/UCF-101_jpg --annotation_path utils/ucf101_01.json --result_path  ucf101_results --dataset ucf101 --n_classes 400 --n_finetune_classes 101 --pretrain_path model_weights/resnet-50-kinetics.pth --ft_begin_index 4 --model resnet --model_depth 50 --resnet_shortcut B --batch_size 128 --n_threads 4 --checkpoint 5

**Comment by zhihuilics** [comment]
@kenshohara 

Do you mean this command to run the experiments?

python main.py --root_path ~/project/ucf101  --video_path jpg0 --annotation_path ucf101_01.json \
--result_path results/333 --dataset ucf101 --n_classes 101 \
--model resnext --model_depth 101 --resnet_shortcut B --resnext_cardinality 32 --sample_duration 16 \
--resume_path /home/bernie/project/ucf101/pretrained/resnext-101-kinetics-ucf101_split1.pth \
--batch_size 64 --n_threads 12 --no_train --no_val --test --test_subset val

**Comment by slighting666** [comment]
@zhihuilics 
Hi, I got the accuracy of val.log, but I want to calculate the accuracy of video level, where can I get the val.json file and how to use eval_ucf101.py?
